Cargo.toml

```diff
@@ -23,6 +23,7 @@ async-trait = "0.1"
 aws-lc-rs = { version = "1.8.0", optional = true }
 base64 = "0.22"
 bytes = "1"
+chrono = { version = "0.4", features = ["serde"], optional = true }
```

Comment:
ah, this commit was meant to be squashed into its parent. Will fix.

Reply:
Hmm, also on consideration, rcgen and yasna depend on time instead of chrono today. I wonder if that mean we should use time instead of chrono here?

Reply:
Also, make sure that we have `doc_auto_cfg` set up so `renewal_info()` will be discoverable in the docs even when `cfg`ed out by default.

Reply:
>  I wonder if that mean we should use time instead of chrono here?

Fair point. I switched to using `time`.

> Also, make sure that we have doc_auto_cfg set up so renewal_info() will be discoverable in the docs even when cfged out by default.

I tacked a commit on at the end to try and make this work. Like other repos we have to avoid the `fips` feature but otherwise it seems workable.